### ISSUE TITLE
Website too bright, need dark mode

### ISSUE THREAD

**Issue by vijethx** [wslyvh/useWeb3#51]

**Comment by wslyvh** [comment]
Yes! Agree. Should've made it the default actually. It basically just hurts my eyes 

Will add a new theme once all pages/components have completely switched to the new style.

**Comment by Muhindo-Galien** [comment]
Is there someone working on this issue?

**Comment by Muhindo-Galien** [comment]
that's great
